common/tests/sm_ut/sm_ut.c

```diff
@@ -700,6 +704,29 @@ TEST_FUNCTION(sm_close_begin_with_cb_unhappy_path)
     sm_destroy(sm);
 }
 
+// Tests_SRS_SM_11_002: [ If the state is SM_OPENING, the close_while_opening_callback is non-NULL and this is the first evaluation of the close then ... ]
```

Comment:
> and this is the first evaluation of the close then ... ] [](http://example.com/codeflow?start=98&length=57)

shouldn't we have another test showing the callback is only called once?

Reply:
This test actually goes through the loop twice since we don't call sm_open_end in the callback, so we are ensuring that the callback is only called once.